editor/editor_themes.cpp

```diff
@@ -1121,6 +1121,12 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_constant("margin_bottom", "MarginContainer", 0);
 	theme->set_constant("hseparation", "GridContainer", default_margin_size * EDSCALE);
 	theme->set_constant("vseparation", "GridContainer", default_margin_size * EDSCALE);
+	theme->set_constant("hseparation", "FlowContainer", default_margin_size * EDSCALE);
+	theme->set_constant("vseparation", "FlowContainer", default_margin_size * EDSCALE);
```

Comment:
These 2 constants aren't used, no?

Reply:
They are actually used, as fallback values (theme lookup uses class inheritance to look for parent class definitions for all controls). So it's safe to keep them.